components/enrichers/reachability/cmd/main.go

```diff
@@ -0,0 +1,92 @@
+package main
```

Comment:
If you are not familiar with the concept, code in `internal` [cannot be imported by other go modules](https://dave.cheney.net/2019/10/06/use-internal-packages-to-reduce-your-public-api-surface). The build tool would report an error if this happens - leading to a failing build.

This makes sure to leak to potential user only the API that we actually want to expose. This is a nice Go built in feature